Assets/scripts/enemyController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class enemyController : MonoBehaviour
+{
+    int MoveSpeed = 5;
```

Comment:
Changed this to `public int MoveSpeed;` then the movement speed can be edited it in the editor